eng/config/PublishData.json

```diff
@@ -44,7 +44,6 @@
       "Microsoft.CodeAnalysis.CSharp.EditorFeatures": "vssdk",
       "Microsoft.CodeAnalysis.InteractiveHost": "vssdk",
       "Microsoft.CodeAnalysis.VisualBasic.EditorFeatures": "vssdk",
-      "Microsoft.CodeAnalysis.EditorFeatures.Wpf": "vssdk",
```

Comment:
You need to leave this around since this file still gets used for other feature branches for servicing. It's OK if it's there but the file doesn't exist in main.